wasm/src/main/java/com/dylibso/chicory/wasm/Parser.java

```diff
@@ -339,8 +344,8 @@ private static ImportSection parseImportSection(ByteBuffer buffer) {
 
         // Parse individual imports in the import section
         for (int i = 0; i < importCount; i++) {
-            String moduleName = readName(buffer);
-            String importName = readName(buffer);
+            String moduleName = readName(buffer, false);
```

Comment:
same as the default, we can use the single parameter version, here and the next.